compute/compute-node.Dockerfile

```diff
@@ -1247,6 +1328,7 @@ COPY --from=unit-pg-build /postgresql-unit.tar.gz /ext-src/
 COPY --from=vector-pg-build /pgvector.tar.gz /ext-src/
 COPY --from=vector-pg-build /pgvector.patch /ext-src/
 COPY --from=pgjwt-pg-build /pgjwt.tar.gz /ext-src
+COPY --from=pgrag-pg-build /usr/local/pgsql/ /usr/local/pgsql/
```

Comment:
Since we don't have tests for `pgrag` we don't need to copy it to `neon-pg-ext-test`